staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -623,6 +629,15 @@ func (e *Store) Get(ctx genericapirequest.Context, name string, options *metav1.
 	return obj, nil
 }
 
+// qualifiedResourceFromContext retrieves GroupResource from Context per request, if not found, return DefaultQualifiedResource
```

Comment:
qualifiedResourceFromContext attempts to retrieve a GroupResource from the context's request info.
If the context has no request info, the DefaultQualifiedResource is used.